types/types.go

```diff
@@ -243,6 +243,7 @@ type Info struct {
 	ServerVersion      string
 	ClusterStore       string
 	ClusterAdvertise   string
+	SecurityOptions    string
```

Comment:
should we make this a `[]string`? I think that'd be better for the API (and similar to, e.g. how we have a list of installed plugins; https://github.com/docker/engine-api/blob/0ec73ff0299c6bc44426146a4ecaf5250536ea1c/types/types.go#L248-L257)